codalab/model/tables.py

```diff
@@ -78,7 +79,7 @@
   Column('uuid', String(63), nullable=False),
   Column('name', String(255), nullable=False),
   Column('owner_id', String(255), nullable=True),
-  Column('title', String(255), nullable=True),  # Short human-readable description of the worksheet
+  Column('title', Unicode(255), nullable=True),  # Short human-readable description of the worksheet
```

Comment:
Just out of curiosity, is `Text` unicode by default (what the metadata field values and worksheet items are)?